### ISSUE TITLE
glacier.h5 file empty?

### ISSUE THREAD

**Issue by christopheogier** [eth-vaw-glaciology/GlacioTools.jl#11]
Hello, when running:

```
SGI_ID = "B43-03"
data = fetch_glacier("Rhone", SGI_ID; datadir)
```

I get the following error:

```
KeyError: key "glacier/x" not found

Stacktrace:
 [1] getindex(parent::HDF5.File, path::String; pv::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ HDF5 C:\Users\christophe\.julia\packages\HDF5\TcavY\src\highlevel.jl:63
 [2] getindex(parent::HDF5.File, path::String)
   @ HDF5 C:\Users\christophe\.julia\packages\HDF5\TcavY\src\highlevel.jl:62
 [3] read(parent::HDF5.File, name::String; pv::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ HDF5 C:\Users\christophe\.julia\packages\HDF5\TcavY\src\readwrite.jl:16
 [4] read
   @ C:\Users\christophe\.julia\packages\HDF5\TcavY\src\readwrite.jl:15 [inlined]
 [5] load_elevation(path::String)
   @ GlacioTools C:\Users\christophe\.julia\packages\GlacioTools\kXxvK\src\Rasters_helpers.jl:151
 [6] fetch_glacier(name::String, SGI_ID::String; datadir::String)
   @ GlacioTools C:\Users\christophe\.julia\packages\GlacioTools\kXxvK\src\Alpine_glaciers.jl:32
 [7] top-level scope
   @ In[36]:2
```
It seems that the h5 file `glacier` (in this case "Rhone") is empty and has no field `x`
See below:

https://github.com/eth-vaw-glaciology/GlacioTools.jl/blob/4f3a95e9149fc304040921df9b0c89579e050389/src/Rasters_helpers.jl#L151

Thanks for clarification ;)

Chris

**Comment by luraess** [comment]
This should be fixed by #12

**Comment by luraess** [comment]
Closed by #12
